internal/querynodev2/delegator/delegator.go

```diff
@@ -774,10 +788,71 @@ func (sd *shardDelegator) Close() {
 	sd.lifetime.Wait()
 }
 
+// As partition stats is an optimization for search/query which is not mandatory for milvus instance,
```

Comment:
Whenever there's a loading happend, we try to reload the partition stats with the maximum version